proxy/proxy.go

```diff
@@ -112,21 +112,21 @@ func proxyRequest(w http.ResponseWriter, originalReq *http.Request, proxyClient
 	pathVars := mux.Vars(originalReq)
 	functionName := pathVars["name"]
 	if functionName == "" {
-		writeError(w, http.StatusBadRequest, errMissingFunctionName)
+		httputils.ErrorF(w, http.StatusBadRequest, errMissingFunctionName)
```

Comment:
This hasn't been changed to match the new name,  so I'm not sure how CI is passing? @LucasRoesler